src/Components/Homepage.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import fetchCoins from '../Redux/api';
+import Coin from './coin';
+import { setCoins } from '../Redux/Coins';
+import Navbar from './Navbar';
+
+const Homepage = () => {
+  const coins = useSelector((state) => state.coins);
+  const dispatch = useDispatch();
+  const initialData = async () => {
+    const data = await fetchCoins();
+    dispatch(setCoins(data));
+  };
+  if (coins.length < 1) {
+    initialData();
+    return (
+      <div className="text-center">
+        <div className=" text-center p-4 spinner-grow text-light" role="status" />
+      </div>
+    );
+  }
+
+  return (
+    <section>
+      <Navbar />
+      <div className="m-5 grid md:grid-cols-4 sm:grid-cols-2  grid-cols-2 gap-2 ">
+        {coins.map((coin) => (
+          <Coin key={coin.uuid} coin={coin} />
+        ))}
+      </div>
+    </section>
```

Comment:
- [ ] Great job coming up with a design that really matches the provided Behance design. However note that you are required to stick as much as possible to the provided design this included typography and also colors. Your project currently has a white theme this is perfectly fine. However, to meet all the project requirements, you are supposed to choose between the pink theme or the blue theme. Kindly ensure that you make this change.

**This is how your project theme currently looks like:**

![image](https://user-images.githubusercontent.com/86318284/194068975-01c0e6cc-c49d-49a5-ad4d-2cf4555774cc.png)

**This is one of the desired project themes:**

![image](https://user-images.githubusercontent.com/86318284/194069154-2b8416f4-75d7-4cc5-bb1b-94a0c52c23d6.png)